templates/compose/ente-photos-with-s3.yaml

```
@@ -28,6 +28,12 @@ services:
       - 'ENTE_JWT_SECRET=${SERVICE_REALBASE64_JWT}'
       - 'ENTE_INTERNAL_ADMIN=${ENTE_INTERNAL_ADMIN:-1580559962386438}'
       - 'ENTE_INTERNAL_DISABLE_REGISTRATION=${ENTE_INTERNAL_DISABLE_REGISTRATION:-false}'
+      - 'ENTE_SMTP_HOST=${ENTE_SMTP_HOST:-smtp.gmail.com}'
+      - 'ENTE_SMTP_PORT=${ENTE_SMTP_PORT:-587}'
+      - 'ENTE_SMTP_USERNAME=${ENTE_SMTP_USERNAME}'
+      - 'ENTE_SMTP_PASSWORD=${ENTE_SMTP_PASSWORD}'
+      - 'ENTE_SMTP_EMAIL=${ENTE_SMTP_EMAIL}'
+      - 'ENTE_SMTP_SENDER_NAME=${ENTE_SMTP_SENDER_NAME}'
```

Add Encryption method env
```suggestion
      - 'ENTE_SMTP_SENDER_NAME=${ENTE_SMTP_SENDER_NAME}'
      - 'ENTE_SMTP_ENCRYPTION=${ENTE_SMTP_ENCRYPTION:-tls}'
```